test/integration/targets/ec2_transit_gateway/tasks/main.yml

```diff
@@ -10,10 +10,14 @@
       region: "{{ aws_region }}"
   no_log: yes
 
+- name: generate unique value for testing
+  set_fact:
+    tgw_description: "{{ 65535 | random | to_uuid }}"
```

Comment:
You might want to use `{{ resource_prefix }}` here instead, as that is a random value which is unique to the current test run. It is helpful for being able to identify what test created a resource, especially when run from CI.

Reply:
incorporated the suggestion as a better update.